rollup.config.mjs

```diff
@@ -0,0 +1,25 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+
+export default {
+  input: './lib/index.js',
+  output: [{
+    file: './lib/bundle.mjs',
```

Comment:
Node.js doesn't support importing esm files without a file extension, and TS doesn't support adding file extension to emitted files, so we still need an mjs bundle.
https://github.com/microsoft/TypeScript/issues/16577